bedrock/firefox/templates/firefox/hello/index.html

```diff
@@ -30,7 +30,7 @@
 {% block site_header_unwrapped %}
   {% call fxfamilynav('features') %}
     <a class="try-hello-button" href="https://support.mozilla.org/kb/where-firefox-hello-button">{{ _('Try Hello now') }}</a>
-    <a class="nav-download-button button-flat-dark" href="{{ url('firefox.new') }}">{{ _('Download now') }}</a>
+    <a class="nav-download-button button-flat-dark" href="{{ url('firefox.new') }}">{{ _('Download') }}</a>
```

Comment:
How do you folks feel about keeping "Download now" for l10n? "Download" is a problematic word in English, since it can be both noun and verb, and there's now way to differentiate between the two of them in lang files.